adapters/resetdigital/resetdigital.go

```diff
@@ -0,0 +1,274 @@
+package resetdigital
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"strconv"
+	"text/template"
+
+	"github.com/prebid/openrtb/v20/openrtb2"
+	"github.com/prebid/prebid-server/v2/adapters"
+	"github.com/prebid/prebid-server/v2/config"
+	"github.com/prebid/prebid-server/v2/openrtb_ext"
+)
+
+// MaximumBids is the maximum number of bids that can be returned by this adapter.
+const (
+	MaxBids = 1
+)
```

Comment:
Starting var/const with captial letter makes it public and scope of `MaxBids` is limited to  `resetdigital` package. Consider making `MaxBids` private

```
// MaximumBids is the maximum number of bids that can be returned by this adapter.
Const MaxBids = 1
```